lib/config/untyped.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  'extends': './base',
+  'rules': {
+    // Warns for an awaited value that is not a Promise.
+    'await-promise': false,
+    // Enforces documentation for important items be filled out.
+    'completed-docs': false,
```

Comment:
This isn't a typed rule.  My original version of the list wasn't accurate, and you must've grabbed it faster than I could update :P  Please checkout the comment and update this.